rpc/common/service.proto

```diff
@@ -128,3 +128,10 @@ message CVSS {
   double v2_score  = 3;
   double v3_score  = 4;
 }
+
+message CustomResource {
+  string                 type     = 1;
+  string                 filepath = 2;
+  Layer                  layer    = 3;
+  google.protobuf.Value  info     = 4;
```

Comment:
`data`, `value`, `content` or something like that may be better.

Reply:
I will change it to data